integrations/access/opsgenie/client.go

```diff
@@ -160,7 +160,7 @@ func (og Client) CreateAlert(ctx context.Context, reqID string, reqData RequestD
 
 func (og Client) getResponders(reqData RequestData) []Responder {
 	schedules := og.DefaultSchedules
-	if reqSchedules, ok := reqData.SystemAnnotations[types.TeleportNamespace+types.ReqAnnotationSchedulesLabel]; ok {
+	if reqSchedules, ok := reqData.SystemAnnotations[types.TeleportNamespace+types.ReqAnnotationNotifyServicesLabel]; ok {
```

Comment:
Why did we change the lookup key?
Are we not interested in schedules here?

Reply:
In opsgenie's case both of these annotations are used to store schedules, the change is because the original use of the `/schedules` annotation was a mistake. The responders for an alert should be based on the schedules that should be the notified and the `/schedules` annotation should only be used for the auto approval flow.